update_datenraum_nodes.py

```diff
@@ -1,18 +1,21 @@
 import sys
 import json
-from datetime import datetime
 
 from datenraum import create_datenraum_session
-from convert2rss import get_description
+from enhance_metadata import enhance_metadata
 
-DESCRIPTION_PATH = "public/description-cache.json"
+ENHANCED_METADATA_PATH = "public/enhanced-metadata.json"
 
 
 def main(metadata_file, nodes_file):
+    description_path = "public/description-cache.json"
 
-    published_date = datetime.utcnow()
+    with open(description_path, "r", encoding="utf-8") as input_file:
+        description_cache = json.load(input_file)
+
+    enhance_metadata(metadata_file, description_cache, ENHANCED_METADATA_PATH)
```

Comment:
Can you put this into https://github.com/serlo/metadata-exports/blob/main/download_metadata.py -> so that it only runs one time?